packages/turf-rbush/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@turf/rbush",
```

Comment:
I think we should call this `@turf/geojson-rbush` for consistency and in case we ever want to pull the `rbush` dependency in. (And therefore the directory name may need to be changed too)

Reply:
Sure thing. Will make that change.